Package.swift

```diff
@@ -16,7 +16,7 @@ let package = Package(
     dependencies: [
         .package(
             url: "https://github.com/zksync-sdk/web3swift.git",
-            exact: "2.6.5-zksync"
+            .exact("mm/MTL-153-rename-ergs-to-gas")
```

Comment:
Temporary. https://github.com/zksync-sdk/web3swift/pull/4 must be merged beforehand.